ui/app/templates/components/toolbar-secret-link.hbs

```diff
@@ -0,0 +1,17 @@
+<SecretLink
```

Comment:
I would much prefer to pass these attributes to the SecretLink component directly and just tell it to use the `ToolbarButton` template, but the suggestions that I found to use `layoutName` (or `templateName`) weren't working. Is there a better way to do that now?

Reply:
Oh yes! I think this wasn't quite right with the polyfill for angle brackets, but I've used it elsewhere - if they're HTML attributes, you can just do `...attributes` and they'll get passed down (🤔we may need to update secretLink too, not sure). The actual component attrs will still need to be there though.

Reply:
That would be a big improvement, but it doesn't seem to work. The polyfill also says that folders should work now, which would clean this up a little, but that gives me an error too. I can't see where we have a polyfill in the package lists though (other than babel), am I missing something?

Reply:
We were using it before it was shipped in native and then when we upgraded didn't need it anymore. Since then they've proposed and added the nesting syntax so we could add it back to get that (until we upgrade to 3.9 when it's in ember).

Reply:
Ah I looked at secret-link.hbs and I'm guessing this is a no because we can't use the angle-bracket syntax for built-in helpers yet :( sorry.

Reply:
Ah, dang. Well, glad it's just a "yet" then and we can clean it up later :+1: